build/common.sh

```diff
@@ -203,6 +207,18 @@ function kube::build::is_osx() {
   [[ "$(uname)" == "Darwin" ]]
 }
 
+function kube::build::update_dockerfile() {
+  if kube::build::is_osx; then
```

Comment:
hm, I'm not sure that `("-i ''")` is correct - that's just treating it as a single array element.